src/Glpi/Controller/ErrorController.php

```diff
@@ -160,6 +162,14 @@ private function getErrorResponse(\Throwable $exception, Request $request): Resp
             }
         }
 
+        switch (true) {
+            case $exception instanceof MaintenanceException:
+                return new Response(TemplateRenderer::getInstance()->render('maintenance.html.twig', [
+                    'title'            => "MAINTENANCE MODE",
+                    'maintenance_text' => $CFG_GLPI["maintenance_text"] ?? "",
+                ]));
+        }
+
```

Comment:
It should probably be caught by a dedicated listener. Indeed, if we handle it here, then it will be considered as a real error and will result in unexpected log entries.

Reply:
Done